src/components/projects/projects-table/projects-table.tsx

```diff
@@ -33,7 +86,80 @@ const ProjectsTable: React.FC = () => {
     }) as DisplayProject[];
   };
 
-  return <PrjsTable allProjects={transformToDisplayProjects(data!)} />;
+  /**
+   * Updates state with data from input parameters.
+   * @param status The status of the project.
+   * @param projectLead The project lead of the project.
+   * @param projectManager The project manager of the project.
+   * @param carNumber The carNumber of the project.
+   */
+  const sendDataToParent = (
+    status: string,
+    projectLead: string,
+    projectManager: string,
+    carNumber: string
+  ) => {
+    setStatus(status);
+    setProjectLead(projectLead);
+    setProjectManager(projectManager);
+    setCarNumber(carNumber);
+  };
+
+  /**
+   * Returns an array of user names who are listed as a project's lead.
+   */
+  const getLeads = (): string[] => {
+    const projects = data!;
+    let leads: string[] = [];
+    for (let project of projects) {
+      for (let user of project.projectLead) {
+        let name: string = fullNamePipe(user);
+        if (!leads.includes(name)) {
+          leads.push(name);
+        }
+      }
+    }
+    return leads;
+  };
+
+  /**
+   * Returns an array of user names who are listed as a project's managers.
+   */
+  const getManagers = (): string[] => {
+    const projects = data!;
+    let managers: string[] = [];
+    for (let project of projects) {
+      let name = fullNamePipe(project.projectManager);
+      if (!managers.includes(name)) {
+        managers.push(name);
+      }
+    }
+    return managers;
+  };
+
+  return (
+    <>
+      <PageTitle title={'Projects'} />
+      <div className={styles.container}>
+        <Row>
+          <div className={styles.filterTable}>
+            <ProjectsTableFilter
+              onClick={sendDataToParent}
+              leads={getLeads()}
+              managers={getManagers()}
+            />
+          </div>
+          <div className={styles.projectsTable}>
+            <PrjsTable
+              allProjects={transformToDisplayProjects(
```

Comment:
why not filter first and then transform? that would enable you to have more info for when filtering